package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "parasoft-findings-bitbucket",
+  "version": "1.0.0",
+  "description": "A script for uploading results of Parasoft static analysis report to BitBucket.",
```

Comment:
Change to: A CLI tool to read Parasoft static analysis reports and upload results to Bitbucket.